src/evict/evict_lru.c

```diff
@@ -2458,8 +2463,8 @@ __wt_cache_eviction_worker(WT_SESSION_IMPL *session, bool busy, bool readonly, d
                 if (cache->evict_aggressive_score > 0)
                     --cache->evict_aggressive_score;
                 WT_STAT_CONN_INCR(session, txn_rollback_oldest_pinned);
-                __wt_verbose_debug1(
-                  session, WT_VERB_TRANSACTION, "%s", session->txn->rollback_reason);
+                __wt_verbose_debug1(session, WT_VERB_TRANSACTION,
+                  "eviction worker block reason: %s", session->txn->rollback_reason);
```

Comment:
I do not see the benefit of adding more information here.

In this case, if there happens to be a rollback for a txn, the rollback error is explicitly stated with the reason. Moreover, this is the application worker code.